### ISSUE TITLE
History in managing users

### ISSUE THREAD

**Issue by bhargavibindhu** [acsestech/virtualassociate#45]
The history always shows 2 entries - one by admin and other by Midhun on every transaction. 
Please correct this and the show only the right log entries.

![image](https://cloud.githubusercontent.com/assets/20394163/19963017/80ec1ea0-a1e0-11e6-9886-53da9ec5cba1.png)

**Comment by NaziniSK** [comment]
As  per the user who did transactions it is showing.

**Comment by bhargavibindhu** [comment]
No, What i meant was - for a single transaction always 2 log entries are created. In the above example - One by VASuperAdmin and other by Midhun. Please check why Midhun entry is created for all transactions. 
@sanndheep5005 @mounikaalladi - Let me know if you too face this issue. 
Need your comments.

**Comment by mounikaalladi** [comment]
Yes Mam i also checked yesterday the same issue occured who is Midhun i didnt get ...??

**Comment by sanndheep5005** [comment]
yes mam midhun entry is created for transactions which we created

**Comment by NaziniSK** [comment]
due to security reasons, login related data we are saving to DB by different process , in that case we are using administrator (midhun )login to update that saved data through some automated process . Because of which in history for every record it is showing like this .

**Comment by bhargavibindhu** [comment]
Please correct it.

**Comment by NaziniSK** [comment]
no correction need to be done here , internally admin(midhun ) also performing transactions , every transaction is recorded in history.

**Comment by bhargavibindhu** [comment]
We need to discuss this, let us park it for now

**Comment by mounikaalladi** [comment]
In history some fields are showing undefined.

![image](https://cloud.githubusercontent.com/assets/23167576/20298652/126ab16a-ab3d-11e6-85e5-4984e65a07ce.png)

**Comment by bhargavibindhu** [comment]
There are a few undefined entries seen in the History logs, please fix it.
Screen: https://github.com/acsestech/virtualassociate/issues/45#issuecomment-260580385

**Comment by bhargavibindhu** [comment]
1. Have you removed the log entries - Midhun? Please confirm.
2. Please display the log entries in descending order. The recent activity should be at the top, 
![image](https://cloud.githubusercontent.com/assets/20394163/20825241/d2ce7a92-b888-11e6-842e-fc0809947bd2.png)

**Comment by NaziniSK** [comment]
Midhun is renamed as Sandip

**Comment by sandipkumarnandi** [comment]
May be we will change the name of the Administrator to Administrator instead of Midhun

**Comment by NaziniSK** [comment]
![image](https://cloud.githubusercontent.com/assets/23185953/20960506/c1ea4222-bc86-11e6-83b2-4feb35e6739e.png)
displaying in descending order now

**Comment by bhargavibindhu** [comment]
Rename it as Administrator. And we need to have a look at Admin login as well as per the discussion we had last time.

**Comment by bhargavibindhu** [comment]
Is this done, if yes please share the screenshot.

**Comment by NaziniSK** [comment]
![image](https://cloud.githubusercontent.com/assets/23185953/21492900/4434f478-cc2e-11e6-88d0-d20ef4c2ecd6.png)
